### PR TITLE
Isolating Hazelcast instance for each test

### PR DESCRIPTION
**Pull request by barrantesgerman**
I have made the suggested corrections

Any other recommendations is welcome

### CODE REVIEW

File: pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java [hunk 1]
```diff
@@ -33,19 +33,16 @@
...
     private static final String KEY = "KEY";
     private static final String VALUE = "VALUE";
-    private static HazelcastInstance hazelcastInstance;
-    private static HazelcastInstance hazelcastInstance2;
+    private HazelcastInstance hazelcastInstance;
 
-    @BeforeClass
-    public static void setUpClass() {
+    @Before
+    public void setUpClass() {
```
Comment (decebals) [inline pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java]: `setUp()` ?

File: pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java [hunk 2]
```diff
@@ -33,19 +33,16 @@
...
         hazelcastInstance = Hazelcast.newHazelcastInstance();
-        hazelcastInstance2 = Hazelcast.newHazelcastInstance();
     }
 
-    @AfterClass
-    public static void tearDownClass() {
+    @After
+    public void tearDownClass() {
```
Comment (decebals) [inline pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java]: `tearDown()` ?

Comment (barrantesgerman) [inline pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java]: I will change

**Comment by decebals** [comment]
Test failed

**Comment by barrantesgerman** [comment]
Yes, I will check

**Comment by barrantesgerman** [comment]
I think I found the problem

Use `put` instead of `set` to guarantee the replication in the cluster
Use `remove` instead of `delete` to guarantee the replication in the cluster
Set 2 async backups to guarantee the copy in the unit test

For more information
http://docs.hazelcast.org/docs/3.6/manual/html-single/index.html#backing-up-maps

**Comment by decebals** [comment]
I see that you have fun doing these implementations :smile:  I am sure that you learn a lot with this occasion.

**Comment by barrantesgerman** [comment]
Yes, it is part of what I like about this job, I am forced to learn something new every day

**Comment by barrantesgerman** [comment]
What would you like to be the next `parent`?

**Comment by decebals** [comment]
The next `parent` can be `pippo-content-type-parent`. I prefer to work with short names, something similar with `pippo-abc-parent` but I feel that it's not possible right now (I don't know if we can change content-type with a simple word).

**Comment by barrantesgerman** [comment]
I think the same, but right now can not think of another name

**Pull request merged**